public/inputs.css

```diff
@@ -141,3 +141,36 @@ input[type='checkbox'] {
   -moz-appearance: checkbox;
   appearance: checkbox;
 }
+
```

Comment:
Would suggest ensuring that if there are any other usages of **file-upload** outside of this feature that you ensure they are tested too.